lib.rs

```diff
@@ -5,6 +5,8 @@
 //! Small vectors in various sizes. These store a certain number of elements inline and fall back
 //! to the heap for larger allocations.
 
+#![feature(test)]
```

Comment:
Correct. Let me work on factoring it into a feature which is disabled by default.